.travis.yml

```
@@ -1,10 +1,15 @@
 language: python
+sudo: false
 python:
-    - 2.7
+  - 2.7
 install:
-    - python bootstrap.py
-    - bin/buildout
+  - pip install -U pip setuptools zope.testrunner
+  - pip install -U coveralls coverage
+  - pip install -U -e .
```

```suggestion
  - pip install -U --pre -e .
```

This allows to install beta versions of the required packages.